integration/dictionary.test.ts

```diff
@@ -272,6 +272,24 @@ describe('Integration tests for dictionary operations', () => {
         expectedStringStringRecord
       );
       expect(hitResponse.valueRecord()).toEqual(expectedStringStringRecord);
+
```

Comment:
check out the lines below - in this version the hitOrElse takes 2 lambdas as args, which allows the caller to deal with the return types so we don't have to.